Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/Bindings/ElementTransformations.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Behavioral.Automation.Template.Bindings.Elements;
+using Behavioral.Automation.Elements;
+using Behavioral.Automation.Services;
+using JetBrains.Annotations;
+using TechTalk.SpecFlow;
+
+namespace Behavioral.Automation.Template.Bindings
```

Comment:
![Uploading image.png…]()

Reply:
@andrey-brovko image is broken for me :(

Reply:
Fix namespace, please )

Reply:
Fixed